docs/topics/api-references.topic

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic SYSTEM "https://resources.jetbrains.com/writerside/1.0/xhtml-entities.dtd">
+<topic xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="https://resources.jetbrains.com/writerside/1.0/topic.v2.xsd"
+       id="api-references" title="API references">
+
+    <p>Welcome to the Kotlin API References page. Here you'll find links to the API documentation for the official Kotlin libraries and tools.</p>
+
+    <list columns="2">
+        <li>
+            <a href="https://kotlinlang.org/api/latest/jvm/stdlib/"><b>Standard Library (stdlib)</b></a>
+            <br/>
+            <p>The Kotlin Standard Library provides core functionality for Kotlin programming. It includes essential APIs such as collections, text and string processing, I/O, ranges, sequences, and more, and is available on all supported platforms.</p>
+            <img src="github.svg" width="18" alt="GitHub"/> <a href="https://github.com/JetBrains/kotlin">kotlin</a>
+            <br/>
+        </li>
+        <li>
+            <a href="https://kotlinlang.org/api/latest/kotlin.test/"><b>Test Library (kotlin.test)</b></a>
+            <br/>
+            <p>A multiplatform testing library that provides common testing annotations and utility functions. It supports integration with popular test frameworks on each platform and offers a unified testing experience across the Kotlin ecosystem.</p>
+            <img src="github.svg" width="18" alt="GitHub"/> <a href="https://github.com/JetBrains/kotlin">kotlin</a>
+            <br/>
+        </li>
+        <li>
+            <a href="https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-core/"><b>Coroutines (kotlinx.coroutines)</b></a>
+            <br/>
+            <p>A powerful library for asynchronous programming using Kotlin coroutines. It offers high-level coroutine builders, concurrency primitives, structured concurrency, and integration with reactive libraries and frameworks.</p>
```

Comment:
1. What does it mean for a coroutine builder to be high-level?
2. I'm not sure what counts as concurrency primitives, but atomic values are in the standard library. What the core part of `kotlinx.coroutines` provides is, in the decreasing order of significance, structured concurrency (which includes coroutine builders), thread pools for asynchronous computations, asynchronous streams of values called "flows", channels, and asynchronous synchronization primitives like the mutex and the semaphore. There's also `kotlinx-coroutines-test`, a test framework for coroutine code.